.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: false
+language: java
+matrix:
+  fast_finish: true
+  include:
+  - name: 'Java 8'
+    jdk: oraclejdk8
+#  - name: 'Java 11'
+#    jdk: openjdk11
+
+before_script:
+# required to avoid test failures from `Picked up _JAVA_OPTIONS: ...` message
+- unset _JAVA_OPTIONS
+script:
+- (cd jib-core; ./gradlew --console=plain --fail-fast --info --stacktrace build)
+- (cd jib-plugins-common; ./gradlew --console=plain --fail-fast --info --stacktrace build)
+- (cd jib-gradle-plugin; ./gradlew --console=plain --fail-fast --info --stacktrace build)
+- (cd jib-maven-plugin; ./mvnw -B fmt:check package -U)
```

Comment:
I think just doing `verify` is fine since `fmt:check` is bound to the `validate` phase.